test/k8s-integration/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Someday, it would be nice to have pd and filestore drivers share the same test runner

Reply:
That is true. Drivers should ideally have a uniform way of build image/push to registry/deploy to cluster(e.g make deploy-driver, make build-image, make push-container), we could write a single k/k integration test runner and plugin the driver configs and generate the test-config.yaml and run the storage e2e testsuites. Perhaps something to do if we write more csi drivers.